src/main/java/org/traccar/protocol/GatorProtocolEncoder.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright 2017 - 2019 Anton Tananaev (anton@traccar.org)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.traccar.protocol;
+
+import io.netty.buffer.ByteBuf;
+import io.netty.buffer.Unpooled;
+import org.traccar.BaseProtocolEncoder;
+import org.traccar.Protocol;
+import org.traccar.helper.Checksum;
+import org.traccar.model.Command;
+
+public class GatorProtocolEncoder extends BaseProtocolEncoder {
+
+    public GatorProtocolEncoder(Protocol protocol) {
+        super(protocol);
+    }
+
+    public static final int GET_POSITION = 0x30;
+    public static final int SET_ENGINE_START = 0x38;
+    public static final int SET_ENGINE_STOP = 0x39;
```

Comment:
These are message types. We always define them in the decoder class. And they should be prefixed with `MSG_`. Make sure the naming also matched the protocol documentation.